modes/pipeline.md

```
@@ -1,57 +1,60 @@
-2. Extraer el número del prefijo (e.g., `142-medispend...` → 142)
-3. Nuevo número = máximo encontrado + 1
+1. List all files in `reports/`
+2. Extract the number from the prefix (e.g., `142-medispend...` → 142)
+3. New number = maximum found + 1
 
-## Sincronización de fuentes
+## Source sync
 
-Antes de procesar cualquier URL, verificar sync:
+Before processing any URL, check sync:
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)